classes/01class/exercises/c01-e04/deniseddi/c01-e04.txt

```diff
@@ -0,0 +1,10 @@
+C01-E04
+- Logged in to AWS console, created and named a S3 bucket "exercise04.week1" (bad name, will do better next time). 
```

Comment:
Any accepted name will do, just keep in mind what I told you in class that S3 bucket names are shared across everyone AWS accounts in the globe. So when creating automation for buckets it's always a good practice to add some random string as a prefix to avoid this issue.